embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -0,0 +1,38 @@
+package io.embrace.android.embracesdk.arch.schema
+
+internal object EmbType {
+
+    /**
+     * Keys that track how fast a time interval is. Only applies to spans.
+     */
+    internal enum class Performance : TelemetryType
```

Comment:
We don't have types and description here... I assume it's going to be added later.

Reply:
There are no subtypes now so this is probably fine until we do